src/porepy/geometry/constrain_geometry.py

```diff
@@ -1,30 +1,44 @@
-b""" Module with various functions to constrain a geometry.
+""" Module with various functions to constrain a geometry.
 
 Examples are to cut objects to lie within other objects, etc.
 """
```

Comment:
I had problems with uploading the html. 
Have a suggestion to write:
``from __future__ import annotations``

Reply:
Yes, indeed. This is needed for Python 3.8. 

Incorporated.